variables.tf

```diff
@@ -11,14 +11,18 @@
 # ------------------------------------------------------------------------------
 
 variable "aws_region" {
-  description = "The AWS region to deploy into (e.g. us-east-1)"
+  description = "The AWS Service endpoint to communicate with."
```

Comment:
* If "AWS Service" is capitalized then "endpoint" probably should be too.
* More importantly, why use the the more confusing terminology "service endpoint" instead of the simpler "region"?

Reply:
I was trying to imply that this isn't actually doing anything in a region.  Everything is "global" but it still needs to talk to an endpoint somewhere.